idiomatic_ruby/array_of_strings_symbols.md

```diff
@@ -3,13 +3,13 @@
 **Array of Strings**
 
 ```ruby
-%w[Michael Jonh Mary Stuart]
+%w(Michael Jonh Mary Stuart)
```

Comment:
Is it worth mentioning that *any* matching brackets work, or should a tip/reminder like that go somewhere else?